### PR TITLE
Fixes bug with scores with 0 goals

### PR DESCRIPTION
**Pull request by rodrigouroz**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,10 +56,12 @@ const validateData = (data) => {
   }
 
   if (!data.games.every(game =>
-    (game.home &&
-     game.away &&
-     Number.isInteger(game.home) &&
-     Number.isInteger(game.away))
+    (
+      {}.hasOwnProperty.call(game, 'home') &&
```
Comment (holden-caulfield) [inline src/index.js]: why not `game.hasOwnProperty('home')`? That looks a lot more natural than creating an object instance from scratch

File: test/index.js [hunk 2]
```diff
@@ -124,4 +124,13 @@ describe('qpjs', () => {
     }
     expect(() => qp(data)).to.throw('Invalid team data')
   })
+
+  it('accepts 0 as a valid score', () => {
```
Comment (holden-caulfield) [inline test/index.js]: Thanks for adding a test!!! :-)

**Review by holden-caulfield — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (rodrigouroz) [inline src/index.js]: Because of this: http://eslint.org/docs/rules/no-prototype-builtins

Comment (holden-caulfield) [inline src/index.js]: You are right, let's try `home in game`

**Review by holden-caulfield — APPROVED** [review]

**Pull request merged**